assets/blocks/quiz/question-block/question.scss

```diff
@@ -0,0 +1,48 @@
+
+/**
+ * Sensei question and question block styles shared between frontend and editor.
+ */
+
+.sensei-lms-question__answer-feedback {
+	--color: #DCDCDE;
+	$horizontal-padding: 24px;
+	border: 1px solid var(--color);
+	border-radius: 1px;
+	margin: 12px auto;
+
+	&--correct {
```

Comment:
Maybe we should consider changing this to underscores (__correct)?

Reply:
This is the [BEM syntax for modifiers](http://getbem.com/naming/), which these are (variations of the base block, not another element inside it.)